pkg/cloudprovider/aws/capacity.go

```diff
@@ -112,3 +117,14 @@ func (c *Capacity) GetOperatingSystems(ctx context.Context) ([]string, error) {
 		v1alpha1.OperatingSystemLinux,
 	}, nil
 }
+
+func (c *Capacity) GetCapacityTypes(ctx context.Context) ([]string, error) {
```

Comment:
yeah (I'll repost from above comment): I had it this way in case the cloud provider wanted to dynamically retrieve the possible capacity types. I don't think they'll change very often... unless we would put placement groups into the capacityType umbrella which would require the dynamic lookup.